src/queries/utils/buildQuery.js

```diff
@@ -19,32 +22,34 @@ const buildQuery = (searchIndex) => {
       queryParts.push(query);
     }
 
-    if ((data.sft || data.saft) && data.plain) {
-      const name = capitalizeFirstLetter(data.name);
+    if (!isExcludedSeeFromLimiter) {
+      if ((data.sft || data.saft) && data.plain) {
+        const name = capitalizeFirstLetter(data.name);
```

Comment:
Just suggestion. We already use `lodash` library in this module, so we can use [`upperFirst`](https://lodash.com/docs/4.17.15#upperFirst) instead of `capitalizeFirstLetter`

Reply:
You're right, thanks!